src/components/CodeBlock.stories.tsx

```diff
@@ -1,85 +1,69 @@
-import 'braid-design-system/reset';
 import 'loki/configure-react';
 
-import React, { type ComponentProps } from 'react';
-
-import {
-  type BraidArgs,
-  defaultArgTypes,
-  defaultArgs,
-} from '../storybook/controls';
-import { BraidStorybookProvider, withRouter } from '../storybook/decorators';
+import type { StoryObj } from 'sku/@storybook/react';
 
 import { CodeBlock as Component } from './CodeBlock';
 
 export default {
+  title: 'Standalone/CodeBlock',
+  component: Component,
   args: {
-    braidThemeName: defaultArgs.braidThemeName,
     graphqlPlayground: 'https://manage.developer.seek.com/graphql-explorer',
     language: 'graphql',
-    size: defaultArgs.size,
+    size: 'standard',
     trim: true,
   },
   argTypes: {
-    braidThemeName: defaultArgTypes.braidThemeName,
-    size: defaultArgTypes.size,
+    size: {
+      control: { type: 'radio' },
+      options: ['standard', 'large'],
+    },
   },
-  component: Component,
-  decorators: [withRouter],
-  title: 'Standalone/CodeBlock',
 };
```

Comment:
```suggestion
} satifies Meta<typeof Component>;
```
https://github.com/SEEK-Jobs/ie-ad-selection-panel/blob/master/src/panels/example/Example.stories.tsx#L11